features/antivirus/antivirus.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Antivirus scanning
 
-@notify @file-upload @requires-aws-credentials
+@notify @file-upload @requires-aws-credentials @skip-local
```

Comment:
Also, this job requires a Notify API token as well as AWS credentials, so it should probably have `@requires-credentials` as a tag

Reply:
> My understanding was that we were trying to eliminate the use of `@skip-local`.

I did not know this.

> but if they do that then they should be able to run the test locally?

Nope. Won't work. Unless you've gone and set up an SNS topic hooked up to a bucket with a subscription to `POST` back to your local machine ... which you've ... set up a public proxy tunnel for?

The thing about all of these individual `@requires-` tags is it starts to add an awful lot of things to remember to exclude. It would be cool if we were able to make `~@skip-local` an alias for all of those things, but I don't think we can.

> so it should probably have `@requires-credentials` as a tag.

Good point.

Reply:
> so it should probably have `@requires-credentials` as a tag.

Actually hold on isn't that denoted by the `@notify` tag?

Reply:
My thinking was that we should prefer marking things as `@requires-credentials` instead of `@skip-local`, as per [this](https://trello.com/c/Ry52eQZY/404-running-functional-tests-locally-that-use-notify-etc) tech debt ticket.

In this case however, as it does seem that it will not be possible to run the job locally, we should probably just mark it as `@skip-local`.

```suggestion
@notify @file-upload @skip-local
```

Reply:
I think it should keep `@requires-aws-credentials` to allow someone who wants to run the tests locally *against* e.g. staging is still able to filter them out if they don't have aws credentials at hand.

Reply:
Are the AWS credentials required different for those required for doing file uploads in general?

Reply:
Prrrrobably not - but you've got to remember to do an `aws-auth` and all that faff for something you might not care about. In short, it requires aws credentials, so I'd like for it to be tagged as such.

Reply:
🤔 I'm just wondering if we could just have `@requires-credentials` and change the manual entry to "All jobs that require API tokens or AWS credentials"

Reply:
But they're two different things, don't remove the ability to skip one but not the other.

Reply:
Just trying to think of ways around the proliferation of `@requires-` tags you were pointed out earlier.

Reply:
On the subject of tags, the feature should probably have a tag like "@antivirus" before the first line so you can easily exclude all scenarios for that feature (if there is ever more than one).

Reply:
The proliferation worries me less if both the coarse _and_ fine grained tags are there. The situation I'd rather avoid is having "excluding all the fine tags" be your only option.

Adding a `@antivirus` tag.